plrustc/README.md

```diff
@@ -26,56 +24,26 @@ We have a few independent workspaces. My `rust-analyzer` configuration currently
-
-5. After all that, you can finally install PL/Rust. I think this is done with `cargo pgx install`? Something like that. Someone should clean up this guide honestly.
-
 ## Usage
 
-Similar to `rustc`, `plrustc` is usually not invoked directly, but instead through `cargo`. While a `plrustc`-using wrapper for cargo exists in this repository, it should not be used except for testing/development/etc.
+Similar to `rustc`, `plrustc` is usually not invoked directly, but instead through `cargo`.
 
 ## Details
 
-Some additional details are provided for users who intend to run PL/Rust and plrustc under restricted environments via seccomp and/or selinux. These details are subject to change, which should be noted in documentation.
-
-### Environment variables
-
-Here are some `plrustc`-specific environment variables that PL/Rust or `plrustc` itself may set. This is in addition to the ones mentioned in "Sysroot Configuration" below.
-
-The `PLRUSTC_FLAGS` variable contains flags passed by PL/Rust which are specific to `plrustc` and required for it to function. The `PLRUSTC_PASSTHROUGH` argument is also used to invoke `plrustc` as if it were plain `rustc`, which it may . FIXME(thom): `PLRUSTC_FLAGS` are not used right now.
-
-PL/Rust is likely to invoke `plrustc` as the value of both `RUSTC` and
-`RUSTC_WRAPPER`. FIXME(thom): seems like we might only need to set it as `RUSTC` now? Not sure.
+Some additional details are provided for users who intend to run PL/Rust and plrustc under restricted environments via seccomp and/or selinux. These details are subject to change, although if that occurs it will be noted in the changelog.
 
 ### Sysroot configuration
 
 To locate the Rust sysroot (which should have the installation of `postgrestd`), the following algorithm is used. It is very similar to the algorithm used by clippy, miri, etc. We stop at the first of these that provides a value.
 
-1. If a `-Zplrustc-sysroot=...` argument is present in `PLRUSTC_FLAGS` then this is used. FIXME(thom): this is not true anymore.
-2. If a `--sysroot` argument is provided via normal program arguments, then that value is used.
-3. The runtime environment is checked.
+1. If a `--sysroot` argument is provided via normal program arguments, then that value is used.
+2. The runtime environment is checked.
    1. First, for `PLRUSTC_SYSROOT` and `SYSROOT` in that order of preference.
    2. Then, for rustup: If both `RUSTUP_HOME` and `RUSTUP_TOOLCHAIN` are set, then we will use the path `$RUSTUP_HOME/toolchains/$RUSTUP_TOOLCHAIN` as the sysroot.
-4. If `rustc` is on the path, then `rustc --print sysroot` is invoked and that value is used.
-5. The compile-time environment is checked.
+3. If `rustc` is on the path, then `rustc --print sysroot` is invoked and that value is used.
```

Comment:
I'm guessing that this has the usual interaction in PL/Rust's handler with the GUC configuring the effective PATH, so that this PATH is then what is seen by plrustc?

Reply:
Yes, it is.